build.sbt

```diff
@@ -305,6 +305,8 @@ def nativeImageDockerSettings: Seq[Setting[_]] = dockerSettings ++ Seq(
   // If this is Some(…): run the native-image generation inside a Docker image
   // If this is None: run the native-image generation using a local GraalVM installation
   graalVMVersion := Some(GraalVersion),
+  graalVMContainerBuildImage := graalVMContainerBuildImage.value
+      .map(_ + "-java11"), // We want to use the Java 11 version of GraalVM
```

Comment:
I think this will work, but I don not understand the difference btw. this approach and the one I suggested (set `GraalVersion` to `something-java11`.

Reply:
@hseeberger The Defense presents exhibit A: https://github.com/cloudstateio/cloudstate/blob/master/build.sbt#L836

Reply:
Accepted ;-)

Reply:
Doesn't seem to work — I still see it build with java8 version. The container build image is already created at this point, it just modifies its name. Could just do this instead?

```
graalVMVersion := Some(GraalVersion + "-java11")
```

Reply:
@pvlugter Done!